README.md

```diff
@@ -67,9 +67,15 @@ To hide pagination
 {{< embed-pdf url="./path/to/pdf/file/example.pdf" hidePaginator="true" >}}
 ```
 
+To hide loading spinner
+```
+{{< embed-pdf url="./path/to/pdf/file/example.pdf" hideLoader="true" >}}
```

Comment:
Since there isn't a loader that we show by default, `hideLoader` doesn't sound right.
Can you make it `showLoader` instead?
In the case of the paginator, it is shown by default hence, `hidePaginator` made sense.
let me know your thoughts.

Reply:
Hey, sorry just now seeing this...

Actually this PR will implement a loader that does show by default, much like the pagination. I decided to follow your previous design in this manner, that's why I say `hideLoader` because the loader will be default. 

I think a default loader makes sense because most applications use a loader for content that takes a few seconds to load in. If you don't want the loader to be default then yes - it should be `showLoader` and I'll have to make changes to the JS logic as well.

Reply:
Sounds good @jeremybohannon.
I have been busy with work and haven't been able to spend time on this for a while. 
I will test this PR and shortly cut another long overdue release.